Source/CMakeLists.txt

```diff
@@ -282,6 +282,12 @@ install(
 	DIRECTORY ../Data
 	DESTINATION ${DATA_DST_DIR}
 	PATTERN "../Data/Raw" EXCLUDE
+	PATTERN "../Data/Shaders/Includes" EXCLUDE
 	PATTERN ".version" EXCLUDE
 	PATTERN ".reqversion" EXCLUDE
 )
+
+install(
+    DIRECTORY ../Data/Raw/Shaders/Includes
```

Comment:
Make sure to use the same white space as the rest of the file (tabs in this case).

Reply:
Oh, right. I wasn't expecting there to be tabs in a CMake file... my editor uses two spaces to indent cmake files so I made sure to add extra spaces, but completely missed the tabs